src/manifold/src/manifold.cpp

```diff
@@ -71,6 +73,67 @@ Manifold Halfspace(Box bBox, glm::vec3 normal, float originOffset) {
   float zDeg = glm::degrees(glm::atan(normal.y, normal.x));
   return cutter.Rotate(0.0f, yDeg, zDeg);
 }
+
+double DistanceToVector(const glm::vec3& pt, const glm::vec3& v) {
+  return glm::length(pt - v * glm::dot(pt, v));
+}
+
+inline double DistanceToPlane(const glm::vec3& pt, const glm::vec4& plane) {
+  return glm::dot({plane.x, plane.y, plane.z}, pt) - plane.w;
+}
+
+glm::vec4 PlaneOfPts(const glm::vec3& a, const glm::vec3& b,
+                     const glm::vec3& c) {
+  auto crx = glm::cross(c - a, b - a);
+  auto n = glm::length(crx);
+  if (n == 0.) return glm::vec4();
```

Comment:
I believe it's faster to `glm::normalize` and check `isfinite`. Also, for complex types, `auto` is fine, but I prefer to be explicit with simple ones, and also to put `const` wherever possible.